tests/appsec/test_blocking_addresses.py

```diff
@@ -134,6 +135,20 @@ def test_response_cookies(self):
         assert False
 
 
+def validate_custom_event_tag(expected_value="", present=True):
```

Comment:
It's more for the code clarety, could you split this in twwo functions : 

```python
def _assert_custom_event_tag_presence(expected_value):
def _assert_custom_event_tag_absence():
```

IMO, the test line will be a little more clear, and won't require any cognitive effort on reading nested `if/else`.

Reply:
Done